### ISSUE TITLE
Binance Exchange `split_trading_pair` method not properly imported.

### ISSUE THREAD

**Issue by mauhcs** [CoinAlpha/hummingbot#4297]
**Describe the bug**
// A clear and concise description of what the bug is.
```
from hummingbot.connector.exchange.binance.binance_exchange import BinanceExchange
print(BinanceExchange.split_trading_pair(trading_pair))
```
 returns the wrong base quote pair.

The reason is because `BinanceExchange` class uses the `connector_base` ([link](https://github.com/CoinAlpha/hummingbot/blob/9bf8e0cc16c00e7458185a2a772a46f1b08f56db/hummingbot/connector/connector_base.pyx#L102)) instead of using the already existing function from `binance_utils.py` ([link](https://github.com/CoinAlpha/hummingbot/blob/9bf8e0cc16c00e7458185a2a772a46f1b08f56db/hummingbot/connector/exchange/binance/binance_utils.py#L17))

**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

To reproduce:

```
trading_pair = "BTCETH"
from hummingbot.connector.exchange.binance.binance_exchange import BinanceExchange
print(BinanceExchange.split_trading_pair(trading_pair))
```

**Screenshots**
// If applicable, add screenshots to help explain your problem.

**Release version**
// Include your bot's version number (Can be found at the upper left corner of your CLI).

One can see this issue in this commit, for example: `9bf8e0cc16c00e7458185a2a772a46f1b08f56db`

**Comment by dennisocana** [comment]
Fix included in version 0.44 release

**Comment by nicolas-powertrade** [comment]
Please beware Tuple was not imported from typing library
